Packs/EWS/Integrations/EwsExtensionEXOPowershellV2/EwsExtensionEXOPowershellV2.yml

```diff
@@ -363,15 +278,310 @@ script:
     - contextPath: EWS.Recipient.RecipientTypeDetails
       description: Recipient type details.
       type: String
-    deprecated: false
-    description: Displays existing recipient objects in your organization. This command
-      returns all mail-enabled objects (for example, mailboxes, mail users, mail contacts,
-      and distribution groups).
-    execution: false
+    description: Displays existing recipient objects in your organization. This command returns all mail-enabled objects (for example, mailboxes, mail users, mail contacts, and distribution groups).
     name: ews-recipient-list
-  isfetch: false
+  - name: ews-new-tenant-allow-block-list-items
+    arguments:
+    - name: entries
+      required: true
+      description: Entries to add to the list.  Separate multiple entries with a comma (e.g. "Item1,Item2").
+      isArray: true
+    - name: list_type
+      required: true
+      auto: PREDEFINED
+      predefined:
+      - sender
+      - filehash
+      - url
+      description: List type to add items to.
+    - name: list_subtype
+      auto: PREDEFINED
+      predefined:
+      - Tenant
+      - AdvancedDelivery
+      description: 'List subtype to add items to. '
+      defaultValue: Tenant
+    - name: action
+      required: true
+      auto: PREDEFINED
+      predefined:
+      - Block
+      - Allow
+      description: Action to set for new entries
+    - name: notes
+      description: Notes to include on new list entries
+    - name: expiration_date
+      description: Enter a specific date and time for the new entries to expire using format "YYYY-MM-DD HH:MM:SSz" for UTC time.  Alternately, a PowerShell **GetDate** statement can be used.
```

Comment:
Can we use the ISO format (`YYYY-MM-DDTHH:MM:SSZ`) instead? We are using it across the product.

Reply:
The time format I used came from the Microsoft documentation for this Powershell cmdlet, however I tested the command with ISO formatting and that works as well.  I'm updating the field description to specify to use ISO format.

Done.